cla-backend-go/package.json

```
@@ -31,6 +31,7 @@
   },
   "resolutions": {
     "axios": "^0.30.3",
+    "tar": "^7.5.10",
```

Biome (2.4.6)

[error] The key tar was already declared.

(lint/suspicious/noDuplicateObjectKeys)